### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by ruseinov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +33,24 @@ Breaking changes
...
+- Reduce timeout resolution in client/backend interface #447
+- Move timeout point to expired state (for atomic swaps) #446
+- Simplify validation logic in Handlers #437
+- Charge gas in x/sigs #429
+- New bump sequence Tx #388
+- Anti-spam fee changes    
+  
+Breaking changes  
+  
+- Move Memo and ChainID to top level Tx codec #435
```
Comment (webmaster128) [inline CHANGELOG.md]: This never happened, right?

Comment (ruseinov) [reply CHANGELOG.md]: good catch, just got there from our auto-generated changelog..

**Comment by ruseinov** [comment]
@husio you can go forward and modify this.
As the changelog hasn't been maintained for 0.14 I had no other sources rather than issues and commit messages.

**Comment by webmaster128** [comment]
> I have mixed feelings about it. This is all generated, right?

I also have strong negative feelings against auto generated CHANGELOGs. If you care for your users, there should be moderation of what is included and what is not. Also a lot of implementations details (like ticket numbers) and terms should be hidden from the reader.

Right now I don't trust the current state of this document, as there is no breaking change included between v0.13.0 and v0.14.0, or at least not visible to me.

**Comment by ruseinov** [comment]
> Right now I don't trust the current state of this document, as there is no breaking change included between v0.13.0 and v0.14.0, or at least not visible to me.

@webmaster128 Neither do I, I'd prefer that all the parties involved reviewed this to see what's missing.
The current changelog for 0.14 is just a placeholder for something meaningful, but it's going to take time to assemble as we have no source of truth for that except for codebase and what developers remember :(

**Comment by husio** [comment]
[As requested](https://github.com/iov-one/weave/pull/630#issuecomment-492619442) I am creating the other pull request directly to `master` branch rendering this one obsolete.

**Pull request closed without merging**